Description: don't build icc2ps (instead use psicc from liblcms2-utils)
Author: Michael Gilbert <mgilbert@debian.org>

--- a/Makefile
+++ b/Makefile
@@ -188,13 +188,6 @@
 		foomatic-db/*/*.xml \
 		foomatic-test \
 		getweb.in \
-		icc2ps/*.[ch] \
-		icc2ps/*.1in \
-		icc2ps/Makefile \
-		icc2ps/AUTHORS \
-		icc2ps/COPYING \
-		icc2ps/README \
-		icc2ps/README.foo2zjs \
 		osx-hotplug/Makefile \
 		osx-hotplug/*.m \
 		osx-hotplug/*.1in \
@@ -376,7 +369,7 @@
 # The usual build rules
 #
 all:	all-test $(PROGS) $(BINPROGS) $(SHELLS) getweb \
-	all-icc2ps all-osx-hotplug man doc \
+	all-osx-hotplug man doc \
 	all-done
 
 all-test:
@@ -598,7 +591,7 @@
 #
 # Installation rules
 #
-install: all install-test install-prog install-icc2ps install-osx-hotplug \
+install: all install-test install-prog install-osx-hotplug \
 	    install-extra install-crd install-foo install-ppd \
 	    install-gui install-desktop install-filter \
 	    install-man install-doc
@@ -1130,7 +1123,6 @@
 	-rm -f $(MANDIR)/man1/foo2hbpl*.1 $(MANDIR)/man1/hbpldecode.1
 	-rm -f $(MANDIR)/man1/gipddecode.1
 	-rm -f $(MANDIR)/man1/arm2hpdl.1 $(MANDIR)/man1/usb_printerid.1
-	-rm -f $(MANDIR)/man1/foo2zjs-icc2ps.1
 	-rm -rf /usr/share/foo2zjs/
 	-rm -rf /usr/share/foo2hp/
 	-rm -rf /usr/share/foo2oak/
@@ -1155,7 +1147,6 @@
 	-rm -f /usr/bin/hbpldecode
 	-rm -f /usr/bin/opldecode
 	-rm -f /usr/bin/rodecode
-	-rm -f /usr/bin/foo2zjs-icc2ps
 	-rm -f /usr/bin/foo2zjs-pstops
 	-rm -f /usr/bin/command2foo2lava-pjl
 	-rm -f /usr/lib/cups/filter/command2foo2lava-pjl
@@ -1198,7 +1189,6 @@
 	-rm -f *.zjs *.zm *.zc *.zc? *.zc?? *.oak *.pbm *.pksm *.cmyk
 	-rm -f pksm2bitcmyk
 	-rm -f *.icm.*.ps
-	cd icc2ps; $(MAKE) $@
 	cd osx-hotplug; $(MAKE) $@
 
 #
@@ -1348,7 +1338,7 @@
 #
 #	icc2ps regression tests
 #
-ICC2PS=./icc2ps/foo2zjs-icc2ps
+ICC2PS=/usr/bin/psicc
 icctest:
 	for g in *.icm; do \
 	    for i in 0 1 2 3; do \
@@ -1446,7 +1436,7 @@
 # Manpage generation.  No, I am not interested in "info" files or
 # HTML documentation.
 #
-man: $(MANPAGES) man-icc2ps man-osx-hotplug
+man: $(MANPAGES) man-osx-hotplug
 
 $(MANPAGES): macros.man includer-man
 
@@ -1513,7 +1503,6 @@
 	$(INSTALL) -c -m 644 arm2hpdl.1 $(MANDIR)/man1/
 	$(INSTALL) -c -m 644 usb_printerid.1 $(MANDIR)/man1/
 	$(INSTALL) -c -m 644 printer-profile.1 $(MANDIR)/man1/
-	cd icc2ps; $(MAKE) install-man
 ifeq ($(UNAME),Darwin)
 	cd osx-hotplug; $(MAKE) install-man
 endif
@@ -1534,10 +1523,9 @@
 
 GROFF=/usr/local/test/bin/groff
 GROFF=groff
-manual.pdf: $(MANPAGES) icc2ps/foo2zjs-icc2ps.1 osx-hotplug/osx-hplj-hotplug.1
+manual.pdf: $(MANPAGES) osx-hotplug/osx-hplj-hotplug.1
 	-$(GROFF) -t -man \
 	    `ls $(MANPAGES) \
-		icc2ps/foo2zjs-icc2ps.1 \
 		osx-hotplug/osx-hplj-hotplug.1 \
 		| sort` \
 	    | ps2pdf - $@
